opensearch_py_ml/sentence_transformer_model/sentencetransformermodel.py

```diff
@@ -27,19 +28,48 @@
 
 
 class SentenceTransformerModel:
-    def __init__(self, model_id: str = None) -> None:
+    def __init__(
+        self, model_id: str = None, folder_path: str = None, overwrite: bool = False
+    ) -> None:
         """
         Description:
-        Initiate a sentence transformer model object.
+        Initiate a sentence transformer model object. The model id will be used tp download pretrained model from the
+        hugging-face and served as the default name for model files, and the folder_path will be the default location
+        to store files generated in the following functions.
 
         Parameters
         ----------
         model_id: str = None
-             the huggingface mode id to download sentence transformer model, if None, default as 'sentence-transformers/msmarco-distilbert-base-tas-b'
-        Return
-        ----------
-             None
+             Optional, the huggingface mode id to download sentence transformer model,
+             if None, default as 'sentence-transformers/msmarco-distilbert-base-tas-b'
+        folder_path: str = None
+             Optional, the path of the folder to save output files, such as queries, pre-trained model, after-trained
+             custom model and configuration files. if None, default as "/model_files/" under the current work directory.
+        overwrite: bool = False
+             Optional,  choose to overwrite the folder at folder path. Default as false. So when training different
+             sentence transformer models,it's recommended to give designated folder path per model training. But if the
+             training process get interrupted in between, users can choose overwrite = True to restart the process.
+        Returns
+        -------
+        None
         """
+        default_folder_path = os.path.join(os.getcwd() + "/model_files/")
```

Comment:
```suggestion
        default_folder_path = os.path.join(os.getcwd(), "/model_files/")
```

Refer to https://docs.python.org/3/library/os.path.html#os.path.join

Reply:
so in this [doc](https://docs.python.org/3/library/os.path.html#os.path.join) if make a new folder we will need the '/' before the folder name, But if making a file path, we don't need the '/'. make changes according in the new commit